10-flow-rate-monitor/firmware/README.md

```diff
@@ -0,0 +1,42 @@
+# Flow Rate Sensor
+
+This directory contains Arduino-based firmware for a flow rate measurement
+utility. The firmware source code is identical to the valve-monitor firmware in
```

Comment:
We shouldn’t assume the reader knows what “the valve-monitor firmware” means.

Reply:
Good call.